src/workerd/jsg/jsvalue.c++

```diff
@@ -728,4 +728,40 @@ void JsMessage::addJsStackTrace(Lock& js, kj::Vector<kj::String>& lines) {
   }
 }
 
+size_t JsFunction::length(Lock& js) const {
+  JsObject obj = *this;
+  auto lengthVal = obj.get(js, "length"_kj);
+  KJ_IF_SOME(num, lengthVal.tryCast<jsg::JsNumber>()) {
```

Comment:
Do you have any test that validates a function with Infinity or NaN as a length?

Reply:
Nope, and we really shouldn't need one. V8 handles that for us with `NumberValue()` (used when `value(js)` is called.